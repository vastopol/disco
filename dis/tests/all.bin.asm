AND 010010100000
LD R3, 000010000
TRAP 0x23
LDR 001011000000
ADD 100001111100
BRz 000001000
NOT 001001111111
ADD 001001100001
ADD 001001000000
BRnp 000000001
ADD 010010100001
ADD 011011100001
LDR 001011000000
BRnzp 111110110
LD R0, 000000100
ADD 000000000010
TRAP 0x21
TRAP 0x25
BR 000110000

AND 010010100000
ADD 010010000100
ADD 101101111111
BRzp 111111101
TRAP 0x25

AND 001001100000
AND 100100100000
ADD 100100101010
LD R2, 011111100
LDR 011010000000
ADD 010010100001
ADD 001001000011
ADD 100100111111
BRp 111111011
TRAP 0x25

AND 000000100000
ADD 000000100001
AND 001001100000
ADD 001001111011
AND 011011100000
ADD 011011101010
LD R4, 000001001
LDR 010100000000
ADD 010010000001
BRz 000000101
ADD 100100100001
ADD 011011111111
LDR 010100000000
BRp 111111010
AND 000000100000
TRAP 0x25
ST R0, 100000000

AND 001001100000
ADD 001001101111
LDI R2, 000000110
BRn 000000100
ADD 001001111111
ADD 010010000010
BRn 000000001
BRnzp 111111100
TRAP 0x25
ST R0, 100000000
