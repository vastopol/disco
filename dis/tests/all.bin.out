54A0
2610
F023
62C0
187C
0408
927F
1261
1240
0A01
14A1
16E1
62C0
0FF6
2004
1002
F021
F025
0030

54A0
1484
1B7F
07FD
F025

5260
5920
192A
24FC
6680
14A1
1243
193F
03FB
F025

5020
1021
5260
127B
56E0
16EA
2809
6500
1481
0405
1921
16FF
6500
03FA
5020
F025
3100

5260
126F
A406
0804
127F
1482
0801
0FFC
F025
3100
